app/javascript/store.js

```diff
@@ -0,0 +1,10 @@
+import { createStore, combineReducers } from '@reduxjs/toolkit';
+import greetingReducer from './reducers/greetingReducer';
+
+const rootReducer = combineReducers({
+  greeting: greetingReducer,
+});
+
+const store = createStore(rootReducer);
+
+export default store;
```

Comment:
- [ ] Great work on making the Redux store :+1: but you have not connected your redux setup in your app :fearful: It is clearly mentioned in the project requirements that you need to get the greetings in your redux store and then you would show those greetings in your components :thinking: means redux and your components are connected :blush: Kindly connect this redux store with your app and let your app consume this redux store to fetch the greetings from the API instead of making an API call directly from your greetings components :thinking:

- [ ] Secondly, when you implement the redux store in your components we shall be able to see that you have made your redux store perfectly :+1:

Reply:
I fixed this problem.